### ISSUE TITLE
ansible-playbook 03.docker.yml 时报错

### ISSUE THREAD

**Issue by luokeli59** [gjmzj/kubeasz#195]
hi你好！在执行 ansible-playbook 03.docker.yml 时报错
###############
TASK [docker : 开机启用docker 服务] ************************************************************************************************************************
fatal: [192.168.1.201]: FAILED! => {"changed": true, "cmd": "systemctl enable docker", "delta": "0:00:00.052876", "end": "2018-05-19 00:12:24.804751", "msg": "non-zero return code", "rc": 1, "start": "2018-05-19 00:12:24.751875", "stderr": "Failed to execute operation: File exists", "stderr_lines": ["Failed to execute operation: File exists"], "stdout": "", "stdout_lines": []}
...ignoring
fatal: [192.168.1.202]: FAILED! => {"changed": true, "cmd": "systemctl enable docker", "delta": "0:00:00.042905", "end": "2018-05-19 00:12:25.247945", "msg": "non-zero return code", "rc": 1, "start": "2018-05-19 00:12:25.205040", "stderr": "Failed to execute operation: File exists", "stderr_lines": ["Failed to execute operation: File exists"], "stdout": "", "stdout_lines": []}
...ignoring
changed: [192.168.1.203]
fatal: [192.168.1.200]: FAILED! => {"changed": true, "cmd": "systemctl enable docker", "delta": "0:00:00.031272", "end": "2018-05-19 00:12:27.610195", "msg": "non-zero return code", "rc": 1, "start": "2018-05-19 00:12:27.578923", "stderr": "Failed to execute operation: File exists", "stderr_lines": ["Failed to execute operation: File exists"], "stdout": "", "stdout_lines": []}
...ignoring

###############
验证
运行ansible-playbook 03.docker.yml 成功后可以验证
systemctl status docker 	#成功
journalctl -u docker        # 运行日志成功
docker version               # 203失败，其他成功
docker info                    #203失败，其他成功

个人分析：
 1 因为环境变量设置问题，请问在何处设置？ 2 如何解决这种问题？

**Comment by gjmzj** [comment]
嗯，systemctl enable docker的报错可以忽略
